harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -557,11 +558,12 @@ def _compute_validation_metrics(self) -> workload.Response:
             logging.info("Computing validation metrics for model info trial")
             if self.is_chief:
                 searcher_metric_name = self.env.experiment_config["searcher"]["metric"]
+                metrics = dataclasses.asdict(self.context._model_info)
+                metrics[searcher_metric_name] = 0.0  # searcher metric must be present
```

Comment:
Would setting `metrics[searcher_metric_name] = None` error out here? 0.0 might be a confusing value.

Reply:
I also think it won't be necessary for the `searcher_metric_name` key to exist in the metrics dictionary, ultimately, since it will be coordinated by a custom searcher which can handle heterogeneous metric reporting.